tpot/tpot_mdr_classifier.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+tpot_mdr_classifier_config_dict = {
+
+    # Classifiers
+    
+    'mdr.MDR': {
+    'tie_break': tie_break % 2
```

Comment:
These should be value entries, not preprocessing code. So this line would be, e.g.,

```'tie_break': [0,1],```